plotly/figure_factory/_facet_grid.py

```diff
@@ -108,7 +106,7 @@ def _annotation_dict(text, lane, num_of_lanes, SUBPLOT_SPACING, row_col='col',
         showarrow=False,
         xref='paper',
         yref='paper',
-        text=text,
+        text=str(text),
```

Comment:
why was this change necessary?

Reply:
Context: `layout.annotation.text` has a `string` `valType`. I don't currently autoconvert numbers (or any other Python types) to strings. 

Do we want to support any string coercions?

Reply:
This seems like it should be an addition to the plotly.js schema. I'll make an issue

Reply:
Tracking in https://github.com/plotly/plotly.js/issues/2523